From c1731be30def6d1eb37df02288275ebcd9484183 Mon Sep 17 00:00:00 2001
From: Anton Tikhomirov <av.tikhomirov@samsung.com>
Date: Tue, 26 Nov 2013 10:08:58 -0600
Subject: [PATCH] usb: phy: fsm: protect against multiple inclusions

if this header is included twice, we would have
redefinition build errors. Fix this.

Signed-off-by: Anton Tikhomirov <av.tikhomirov@samsung.com>
Signed-off-by: Felipe Balbi <balbi@ti.com>
(cherry picked from commit d49dd788840ff802421ed7412e967b659fe9ca58)
Signed-off-by: Maladau Mou <maladau.mou@intel.com>
---
 include/linux/usb/otg-fsm.h |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/include/linux/usb/otg-fsm.h b/include/linux/usb/otg-fsm.h
index 32f86a3..200f4d1 100644
--- a/include/linux/usb/otg-fsm.h
+++ b/include/linux/usb/otg-fsm.h
@@ -15,6 +15,9 @@
  * 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
+#ifndef __LINUX_USB_OTG_FSM_H
+#define __LINUX_USB_OTG_FSM_H
+
 #include <linux/spinlock.h>
 #include <linux/errno.h>
 
@@ -237,3 +240,5 @@ static inline int otg_start_gadget(struct otg_fsm *fsm, int on)
 }
 
 int otg_statemachine(struct otg_fsm *fsm);
+
+#endif /* __LINUX_USB_OTG_FSM_H */
-- 
1.7.9.5

